llvm/test/CodeGen/AMDGPU/GlobalISel/llvm.amdgcn.struct.buffer.load.format.ll

```diff
@@ -1,6 +1,7 @@
 ; NOTE: Assertions have been autogenerated by utils/update_mir_test_checks.py
 ; RUN: llc -global-isel -mtriple=amdgcn-mesa-mesa3d -mcpu=fiji -stop-after=instruction-select -verify-machineinstrs -o - %s | FileCheck -check-prefix=GFX8 %s
 ; RUN: llc -global-isel -mtriple=amdgcn-mesa-mesa3d -mcpu=gfx1200 -stop-after=instruction-select -verify-machineinstrs -o - %s | FileCheck -check-prefix=GFX12 %s
+; Note that TFE instructions don't have the result initialization to zero due to stopping before finalize-isel - which is where that's inserted
```

Comment:
Maybe change it to stop-after=finalize-isel?

Reply:
I tried that, but that ended up making the output quite different for the rest of the testing.
Some of the other tests verify the zero initialization for global-isel so i thought it was easier to leave these as-is and add the note.

I'm somewhat ambivalent - so I'm happy to do whatever you would prefer.

Reply:
Seems fine as-is.